### PR TITLE
Modernize the CurrentSession service

### PR DESCRIPTION
**Pull request by Windvis**
This converts the CurrentSession service to a native class and removes all async properties since all they are guaranteed to be set once the application boots.

### CODE REVIEW

File: app/services/current-session.js [hunk 1]
```diff
@@ -1,47 +1,42 @@
...
+
+  get canSign() {
+    return this.hasRole('GelinktNotuleren-ondertekenaar');
+  }
 
-export default Service.extend({
-  session: service('session'),
-  store: service('store'),
   async load() {
     if (this.session.isAuthenticated) {
```
Comment (Windvis) [inline app/services/current-session.js:23]: Should I prevent this from running more than once and show a warning if it does?

Comment (nvdk) [reply app/services/current-session.js:23]: I guess that could be a nice addition, but definitely not a requirement

File: app/components/meeting-form.hbs [hunk 2]
```diff
@@ -4,7 +4,7 @@
       <AuLink @linkRoute="inbox.meetings" @skin="secondary" @icon="arrow-left" @iconAlignment="left">
         {{t "meetingForm.backButton"}}
       </AuLink>
-      <span class="au-c-app-chrome__entity">{{get (await this.currentSession.group) 'classificatie.label'}} {{get (await this.currentSession.group) 'naam'}}</span>
+      <span class="au-c-app-chrome__entity">{{this.currentSession.groupclassificatie.label}} {{this.currentSession.group.naam}}</span>
```
Comment (nvdk) [inline app/components/meeting-form.hbs:7]: missing a `.` here, should be `this.currentSession.group.classificatie.label`

**Review by nvdk — CHANGES_REQUESTED** [review]
a small (but critical :) ) typo, but other than that looks ok!

File: app/components/meeting-form.hbs (hunk 2 above)
Comment (Windvis) [reply app/components/meeting-form.hbs:7]: Woops, will fix!

Comment (Windvis) [reply app/components/meeting-form.hbs]: Should be good now.

File: app/services/current-session.js (hunk 1 above)
Comment (Windvis) [reply app/services/current-session.js:23]: Yea it's probably overkill since that will be noticed during PR's anyway.

**Pull request merged**